### PR TITLE
Image pane component integration into backend

### PR DESCRIPTION
**Pull request by drbalar**
If you are running this..
- I added a another chair to scene 3 (classroom). It is at the front of the classroom.
- You will need to update the database as mockdata has been changed.

Also updated component docs with visual pane mockdata info

### CODE REVIEW

File: docs/components-README.md [hunk 1]
```diff
@@ -82,3 +82,33 @@ Simple documentation for components being used and a sample of their `animations
   ...
 }
 ```
+
+## Visual Pane
+
+- `animations_json` - object
+  - `blackboardData`- object
+    - `componentType` - Needs to be the string "visual-pane"
```
Comment (ahmedhamodi) [inline docs/components-README.md:90]: Can we list the possible values for componentType somewhere in this readme?

**Comment by ahmedhamodi** [comment]
Looks good to me - thanks for readme update.

Def want Jay to review still before approving.

File: docs/components-README.md (hunk 1 above)
Comment (jaydhulia) [reply docs/components-README.md:99]: This is a good idea; once we have all the components we need, I will add an enum validation to the backend for ensuring it is only one of those when objects are modified.

File: app/public/static/js/components/visual-pane.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      if (data.hasOwnProperty("position")) {
+        el.setAttribute("position", {
+          x: data.position[0],
+          y: data.position[1],
+          z: data.position[2],
+        });
+      } else {
+        el.setAttribute("position", { x: 0, y: 0, z: 0 });
+      }
+      imageEl.setAttribute("class", "link");
```
Comment (jaydhulia) [inline app/public/static/js/components/visual-pane.js:33]: The imageEl shouldn't be clickable :)

**Review by jaydhulia — APPROVED** [review]
One small comment, but looks great to me! Nice work @drbalar

**Review by jaydhulia — APPROVED** [review]

**Pull request merged**